plugin/src/main/java/org/opensearch/ml/action/register/TransportRegisterModelAction.java

```diff
@@ -169,7 +169,11 @@ protected void doExecute(Task task, ActionRequest request, ActionListener<MLRegi
         if (FunctionName.isDLModel(registerModelInput.getFunctionName()) && !mlFeatureEnabledSetting.isLocalModelEnabled()) {
             throw new OpenSearchStatusException(LOCAL_MODEL_DISABLED_ERR_MSG, RestStatus.BAD_REQUEST);
         }
-        if (registerModelInput.getUrl() != null && !isModelUrlAllowed) {
+        if (registerModelInput.getUrl() != null
+            && !isModelUrlAllowed
+            && registerModelInput.getFunctionName() != FunctionName.METRICS_CORRELATION
+            && !registerModelInput
+                .getUrl().equals("https://artifacts.opensearch.org/models/ml-models/amazon/metrics_correlation/1.0.0b2/torch_script/metrics_correlation-1.0.0b2-torch_script.zip")) {
```

Comment:
let's create a method for:

```
&& registerModelInput.getFunctionName() != FunctionName.METRICS_CORRELATION
            && !registerModelInput
```

the idea is if `registerModelInput.getUrl() != null && !isModelUrlAllowed` or if `registerModelInput.getUrl() != null && it is metrics correlation model`